Libraries/Components/TextInput/TextInput.js

```diff
@@ -284,7 +284,6 @@ var TextInput = React.createClass({
     clearTextOnFocus: PropTypes.bool,
     /**
      * If true, all text will automatically be selected on focus
-     * @platform ios
```

Comment:
I think you mean to remove this line from the comments above.  You removed it from the `selectTextOnFocus` property